rpm/clearwater-monit.spec

```diff
@@ -24,6 +24,7 @@ build_files_list > clearwater-monit.files
 if [ "$1" == 1 ] ; then
   /usr/share/clearwater/clearwater-monit/install/clearwater-monit.postinst
 fi
+chmod 600 /etc/monit/monitrc
```

Comment:
We do - it's done at https://github.com/Metaswitch/clearwater-monit/blob/master/debian/rules#L19